docs/_articles/en/apex/replay-debugger.md

```diff
@@ -51,7 +51,7 @@ Before you generate a debug log for replay debugging, set breakpoints and checkp
         - Or, to convert an existing breakpoint into a checkpoint, right-click the breakpoint, and select **Edit Breakpoint** \| **Expression**. Set the expression to `Checkpoint`.
     1.  To upload your checkpoints to your org to collect heap dump information, open the command palette, and run **SFDX: Update Checkpoints in Org**.
 
-### Set Up an Apex Replay Debugger Session for a Scratch Org or a Default Development Org
+### Set Up an Apex Replay Debugger Session for a Scratch Org or a Developer Edition Org
```

Comment:
FYI @sbudhirajadoc small doc update on the doc site

Reply:
the change LTGM.